torchgen/gen_functionalization_type.py

```diff
@@ -431,7 +431,8 @@ def emit_view_functionalization_body(
         {reverse_lambda.decl()} {{
           return {reverse_lambda.inner_call()}
         }},
-        /*is_multi_output=*/{str(is_multi_output_view).lower()}
+        /*is_multi_output=*/{str(is_multi_output_view).lower()},
+        /*is_as_strided=*/{str(str(f.func.name) == 'as_strided').lower()}
```

Comment:
man, this discount c++ boolean printing LOL

Reply:
yeah... lol